gui/wxpython/gui_core/pyedit.py

```diff
@@ -573,7 +573,7 @@ def OnSetOverwrite(self, event):
             self.parent.overwrite = False
 
 
-class PyEditFrame(wx.Frame):
+class PyEditFrame(wx.Frame, FileHistory):
```

Comment:
Is inheritance how wx.FileHistory is supposed to be used? This seems to me that it should use composition not inheritance. The PyEditFrame is not FileHistory. It is a wx.Frame, but it merely uses FileHistory.

Reply:
Yes of course you are right.